UNIXFS.md

```diff
@@ -43,9 +43,10 @@ message Data {
 	optional bytes Data = 2;
 	optional uint64 filesize = 3;
 	repeated uint64 blocksizes = 4;
-
 	optional uint64 hashType = 5;
 	optional uint64 fanout = 6;
+	optional uint32 mode = 7;
+	optional int64 mtime = 8;
```

Comment:
These are here and not in the `Metadata` message because it's deprecated so let's stay well away from it.